src/main/java/org/ow2/proactive/catalogclient/service/RemoteObjectService.java

```diff
@@ -25,35 +25,52 @@
  */
 package org.ow2.proactive.catalogclient.service;
 
-import org.ow2.proactive.catalogclient.exception.FailedRequestException;
-import org.springframework.stereotype.Service;
-import org.springframework.web.client.RestClientException;
-import org.springframework.web.client.RestTemplate;
+import static io.restassured.RestAssured.given;
 
+import org.ow2.proactive.catalogclient.util.ResponseUtils;
+
+import io.restassured.response.Response;
 import lombok.extern.log4j.Log4j2;
 
 
 /**
  * This class enables to send request to remote server
  */
-@Service
 @Log4j2
 public class RemoteObjectService {
 
+    private final static RemoteObjectService REMOTE_OBJECT_SERVICE = new RemoteObjectService();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AModifiersOrderCheck)